### PR TITLE
crash.subsystem.filesystem: fix mount command for SLE12-SP3 kernel

### PR DESCRIPTION
**Pull request by luis-henrix**
'mnt_root' access in recent kernels is done through the struct vfsmount
'mnt' field.

Signed-off-by: Luis Henriques <lhenriques@suse.com>

### CODE REVIEW

File: crash/subsystem/filesystem/mount.py [hunk 1]
```diff
@@ -126,6 +126,15 @@ def mount_super(mnt):
         return sb
 
     @export
+    @staticmethod
```
Comment (jeffmahoney) [inline crash/subsystem/filesystem/mount.py]: This should be handled as a callback where we have two versions, one that returns mnt['mnt_root'] and the other that returns mnt['mnt']['mnt_root'].  See crash.types.task.pick_get_rss for an example.

**Review by jeffmahoney — CHANGES_REQUESTED** [review]

File: crash/subsystem/filesystem/mount.py (hunk 1 above)
Comment (luis-henrix) [reply crash/subsystem/filesystem/mount.py]: Hmm, ok that does look a bit better.  I'll send you another pull request later, thanks

**Comment by luis-henrix** [comment]
I've re-pushed the mount-fix branch again.  It actually fixes the issue in a different way, as I realised that the for_each_mount() actually returns 'struct mount' in 12SP3, while it was returning 'struct vfsmount' in older kernels.

**Comment by luis-henrix** [comment]
Ok, I've pushed v3 into branch mount-fix-v3.  It's a bit more generic fix, which allows to also fix the '-f' option.

**Pull request closed without merging**